avsomr/avsomrdrawer.cpp

```diff
@@ -0,0 +1,215 @@
+#include "avsomrdrawer.h"
+
+#include <QPainter>
+
+#include "libmscore/score.h"
+#include "libmscore/measure.h"
+
+#include "avslog.h"
+
+using namespace Ms::Avs;
+
+AvsOmrDrawer::AvsOmrDrawer()
```

Comment:
better in a .h?

Reply:
https://github.com/musescore/MuseScore/pull/5692#discussion_r386407502